tutorials/joint_controller.md

```diff
@@ -591,28 +592,28 @@ This is how the model will look:
 Append the following lines before `</model>` tag in the SDF file.
 
 ```xml
- <plugin
- filename="gz-sim-joint-trajectory-controller-system"
- name="gz::sim::systems::JointTrajectoryController">
- <joint_name>RR_position_control_joint1</joint_name>
- <initial_position>0.7854</initial_position>
- <position_p_gain>20</position_p_gain>
- <position_i_gain>0.4</position_i_gain>
- <position_d_gain>1.0</position_d_gain>
- <position_i_min>-1</position_i_min>
- <position_i_max>1</position_i_max>
- <position_cmd_min>-20</position_cmd_min>
- <position_cmd_max>20</position_cmd_max>
-
- <joint_name>RR_position_control_joint2</joint_name>
- <initial_position>-1.5708</initial_position>
- <position_p_gain>10</position_p_gain>
- <position_i_gain>0.2</position_i_gain>
- <position_d_gain>0.5</position_d_gain>
- <position_i_min>-1</position_i_min>
- <position_i_max>1</position_i_max>
- <position_cmd_min>-10</position_cmd_min>
- <position_cmd_max>10</position_cmd_max>
+<plugin
+  filename="gz-sim-joint-trajectory-controller-system"
+  name="gz::sim::systems::JointTrajectoryController">
+  <joint_name>RR_position_control_joint1</joint_name>
+  <initial_position>0.7854</initial_position>
+  <position_p_gain>20</position_p_gain>
+  <position_i_gain>0.4</position_i_gain>
+  <position_d_gain>1.0</position_d_gain>
+  <position_i_min>-1</position_i_min>
+  <position_i_max>1</position_i_max>
+  <position_cmd_min>-20</position_cmd_min>
+  <position_cmd_max>20</position_cmd_max>
+  <topic>topic_name</topic>

```

Comment:
So what's the behavior without `<topic>` specified? Can we make sure that scenario works as well?

Reply:
I guess my question is, has the default topic changed? I ask because https://github.com/gazebosim/gz-sim/pull/2546 changed some default topics for some systems and those systems had to be updated.

Reply:
No, the default topic I don't believe has changed, the tutorial shows to publish this to control it:
https://github.com/gazebosim/gz-sim/blob/76305dfe9065db3b17dfa562c2ee8e6bafa7b1a3/tutorials/joint_controller.md?plain=1#L622

Note that you need to have the topic name match that topic or it won't work.

And then it says to echo:
https://github.com/gazebosim/gz-sim/blob/76305dfe9065db3b17dfa562c2ee8e6bafa7b1a3/tutorials/joint_controller.md?plain=1#L670

Which once again if you change it to listen to published topic won't work. So one or the other needs to be changed/fixed to work.

Reply:
So @azeey do you want an example with only the default topic, only the "assigned topic" or one with both?

Reply:
@azeey , any final decision here?